terraform/modules/cloudfront_distributions/main.tf

```diff
@@ -60,7 +60,7 @@ module "auth_waf" {
   per_ip_rate_limit              = var.auth_waf_per_ip_rate_limit
   excluded_rules                 = ["CrossSiteScripting_BODY"]
   ip_allowlist                   = var.api.ip_allowlist
-  ip_allowlist_uri_path_regex    = ["^/keycloak/", "/realms/delta/", "^/delta-api/"]
```

Comment:
If we're keeping the /keycloak redirect a bit longer, I think we keep this path in the ip allowlist config.